CHANGELOG.md

```diff
@@ -17,7 +17,11 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Bugfixes
 
-* None.
+* Fix a call to `commitWrite(withoutNotifying:)` committing a transaction that
+  would not have triggered a notification incorrectly skipping the next
+  notification.
+* Fix collection notifications missing move information for non-end insertions
```

Comment:
I see. Even though we don't expose a way to do so directly in our APIs, it's possible this would happen when applying sync changes, correct? If so, perhaps this phrasing would be more appropriate?

> Fix changes applied from the synchronization mechanism missing move information in some cases.